functions/zoom-webhook.py

```diff
@@ -159,6 +159,12 @@ def handler(event, context):
 
 
 def update_zoom_status(zoom_event, payload, zip_id):
+
+    try:
+        validate_payload(payload, recording_completed=False)
+    except BadWebhookData as e:
+        return resp_204(f"Ignore {zoom_event}. {e}")
```

Comment:
Or return 400? I've just seen "meeting.ended" come in semi-frequently without a "start_time" and I don't want to be spending time chasing down unimportant errors.